manifests/init.pp

```diff
@@ -266,6 +266,16 @@
   }
   include pulpcore::plugin::certguard
 
+  class { 'foreman_proxy::plugin::pulp':
+    enabled              => false,
+    pulpnode_enabled     => false,
+    pulpcore_enabled     => true,
+    pulpcore_mirror      => $pulpcore_mirror,
+    pulpcore_api_url     => "https://${servername}/${pulpcore::apache::api_path}",
```

Comment:
@ekohl open to suggestions on calculating this, but these are dependent upon how Apache is configured within this module and the hard-coded paths inside the pulpcore module itself

Reply:
I think that's ok and we can consider that as a public API. Only nit I have is that the variables that you use already start with `/` so you end up with `//` now.
```suggestion
    pulpcore_api_url     => "https://${servername}${pulpcore::apache::api_path}",
```

(same below with content)